app/addons/documents/routes-doc-editor.js

```diff
@@ -17,13 +17,57 @@ define([
   './resources',
   '../databases/base',
   './doc-editor/actions',
-  './doc-editor/components.react'
+  './doc-editor/components.react',
+
+  './rev-browser/rev-browser.actions',
+  './rev-browser/rev-browser.components.react'
 ],
 
-function (app, FauxtonAPI, Helpers, Documents, Databases, Actions, ReactComponents) {
+(app, FauxtonAPI, Helpers, Documents, Databases, Actions, ReactComponents,
+RevBrowserActions, RevBrowserComponents) => {
+
+
+  const RevBrowserRouteObject = FauxtonAPI.RouteObject.extend({
+    layout: 'doc_editor',
+    disableLoader: true,
+    selectedHeader: 'Databases',
+    roles: ['fx_loggedIn'],
+
+    routes: {
+      'database/:database/:doc/conflicts': 'revisionBrowser'
+    },
+
+    initialize: function (route, masterLayout, options) {
+      const databaseName = options[0];
+
+      this.docId = options[1];
+      this.database = this.database || new Databases.Model({ id: databaseName });
+      this.doc = new Documents.Doc({ _id: this.docId }, { database: this.database });
+    },
 
+    crumbs: function () {
+      const previousPage = Helpers.getPreviousPageForDoc(this.database, this.wasCloned);
+      const docUrl = FauxtonAPI.urls('document', 'app', this.database.safeID(), this.docId);
+
+      return [
+        { type: 'back', link: previousPage },
+        { name: this.docId + ' > Conflicts', link: '#' }
```

Comment:
Be nice to style this a little better, either by wrapping the `>` in a span and toning down the colour a bit, or maybe changing it to a `&raquo;` & doing the same style. But whatever - low priority, we can change it down the road.